packages/poi-detail/src/area-names.tsx

```diff
@@ -20,11 +20,12 @@ export default function AreaNames({
   vicinity,
   arrowAction,
 }: {
-  areas: Area[]
+  areas: Area[] | string
   vicinity?: string
   arrowAction?: ReactNode
 }) {
-  const names = areas[0]?.name || vicinity
+  const names =
+    (areas && Array.isArray(areas) ? areas[0]?.name : areas) || vicinity
```

Comment:
`areas`는 require props다보니 아래와 같이 수정할 수 있을 듯 해요!

```suggestion
    Array.isArray(areas) ? areas[0]?.name : areas) || vicinity
```